airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/source.py

```diff
@@ -87,17 +95,21 @@ def streams(self, config: Mapping[str, Any]) -> List[Type[Stream]]:
         """
         config: ConnectorConfig = ConnectorConfig.parse_obj(config)  # FIXME: this will be not need after we fix CDK
         api = API(account_id=config.account_id, access_token=config.access_token)
+        if not config.end_date:
+            config.end_date = pendulum.now()
+
```

Comment:
```suggestion
```

Reply:
@keu the default value is None, if we remove this part we're going to send None to pendulum.instance when creating the object. WDYT?

Reply:
not if you use default_factory